live-assignments/anagram/exercise/README.md

```diff
@@ -0,0 +1,32 @@
+# Anagram
+
+## Duration ⏰ 30 minutes max
+
+## Description
+
+Write a function that takes two words as arguments and returns `true` if they are anagrams (contain the exact same letters) and `false` otherwise.  This function needs to be tested and included in the web application provided as part of the exercise.
+
+The solution should also handle edge cases such as empty strings and `undefined` arguments in a sensible way.
+
+The solution should also be commutative, or, in other words, the result of the check must be the same regardless of which way round the words are.
```

Comment:
nit. This sentence is probably not necessary. If the solution is not commutative, then it's not correct

Reply:
I agree, but this was more of a hint to encourage candidates to write a test for commutativity (if that's a word!).